### PR TITLE
Add `.env.sample` file

### LINKED ISSUE #142: Consider adding a `.env.sample` file
Our [npm scripts](https://github.com/rapidsai/node-rapids/blob/main/package.json#L26-L31) create a blank `.env` file when running the `Docker` commands, but users may not know what variables they can place in this file.

To help with this, adding a `.env.sample` file to the repo may be useful. This file could also include comments that describe what each variable does. An example file may look something like this:

```ini
PARALLEL_LEVEL=8 # Sets the number of processor cores to use for builds (?)
NODE_ENV=production # Enables production build optimizations when building TypeScript modules
```

### PR DESCRIPTION
**Pull request by ajschmidt8**
This PR adds a `.env.sample` file with an associated FAQ in the README.md.

Closes #142

### CODE REVIEW

**Comment by bryevdv** [comment]
IIRC the final suggestion was to include comments in the `.env` file to explain the options?

**Comment by mzegar** [comment]
Also, wanted to add, I ran into complication errors while attempting to build `cudf` from the [cudf repo](https://github.com/rapidsai/cudf). Would it be worth to include similar documentation there?

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# docker-compose.yml variables
+
+CCACHE_VERSION=4.1
+CMAKE_VERSION=3.20.2
+CUDAARCHS=72-real
+CUDA_VERSION=11.0
+NODE_VERSION=15.14.0
+PARALLEL_LEVEL=8
+YARN_VERSION=1.22.5
```
[the remark this answers is not in the document]
Comment (ajschmidt8) [reply .env.sample:9]: @trxcllnt, sounds good. should I use this verbatim?

[the remark this answers is not in the document]
Comment (trxcllnt) [reply .env.sample:9]: What @exactlyallan said. The default PARALLEL_LEVEL if unspecified is 1, but you can push that to 4-6 if you have 32GB of memory.

Comment (trxcllnt) [inline .env.sample:9]: Removing the ones I don't recommend people change:

```suggestion
CUDA_VERSION=11.0
PARALLEL_LEVEL=8
```

They _could_ change them, but we won't have tested them. And some like `YARN_VERSION` are tied to whatever ships in node's container, so it's implicitly tied to the `NODE_VERSION`.

File: README.md [hunk 2]
```diff
@@ -64,6 +64,12 @@ After you build the modules, run `yarn demo` from the command line to choose the
 
 You can review [BINDINGS.md](https://github.com/rapidsai/node-rapids/blob/main/BINDINGS.md) to see which bindings have been completed for each of the RAPIDS libraries.
 
+## FAQs
+
+### What is the `.env` file used for and what variables can be set in it?
+
+The `.env` file is used by the [docker-compose.yml](/docker-compose.yml) file when building the development containers (i.e. `yarn docker:build:devel`). Any variables in the [docker-compose.yml](/docker-compose.yml) file can be set in the `.env` file. The [.env.sample](/.env.sample) file includes some common variables that may be set in the `.env` file.
```
Comment (trxcllnt) [inline README.md:71]:
```suggestion
The `.env` file is used by the [docker-compose.yml](/docker-compose.yml) file when building and running the development containers (i.e. `yarn docker:build:devel` and `yarn docker:run:devel`). Any variables in the [docker-compose.yml](/docker-compose.yml) file can be set in the `.env` file. The [.env.sample](/.env.sample) file includes some common variables that may be set in the `.env` file.
```

Comment (ajschmidt8) [reply README.md]: added in 860ce50

**Review by AjayThorve — APPROVED** [review]
Awesome!

**Review by trxcllnt — APPROVED** [review]

**Pull request merged**